asn1scala/src/main/scala/asn1scala/asn1jvm.scala

```diff
@@ -30,7 +30,7 @@ val MASK_POS_INT    = 0x7F_FF_FF_FFL
 opaque type UByte = Byte
 object UByte {
     @inline def fromRaw(u: Byte): UByte = u
```

Comment:
How is it suppose to work? What if the `Byte` value is negative? Especially true for the `toRaw`. @mario-bucev

Reply:
"Raw" types are just aliases, so I don't understand the point of having them. Do you know @mario-bucev ?

Reply:
Therefore, stainless can't reason about them as "unsigned", so they are signed under the hood. So we should either don't have them, or enforce in some way their "unsigned" nature.

Reply:
`UByte`, `UInt` etc. are opaque so outside of `asn1jvm.scala`, these are considered as different types by the compiler. This allows us to define operations such as `<=` for which signed and unsigned behavior do not agree.
`fromRaw` functions takes the bitwise signed representation and turns it into "unsigned" (e.g. `UByte.fromRaw(-1)` is interpreted as 255 unsigned, and `UByte.fromRaw(0) <= UByte.fromRaw(-1)`, whereas this would not hold if we used signed `<=`).

Reply:
Ah okay, I see!
 Thanks :)